nservicebus/messaging/send-a-message.md

```diff
@@ -1,43 +1,54 @@
 ---
-title: Sending a Message
-summary: Describes how to send a message
+title: Sending messages
+summary: Describes how to send messages
 tags: []
 redirects:
 - nservicebus/how-do-i-send-a-message
 ---
 
-Message sending involves using Send operation that takes an argument of a message instance to be delivered. The details differ slightly between version of NServiceBus, as shown in following snippets.
+NServiceBus supports sending different types of messages (see [Messages, Events and Commands](messages-events-commands.md)) to any endpoint. When a message arrives at an endpoint it goes through a pipeline of processing steps. Messages can be sent from outside or inside the message processing pipeline:
 
-Here's how a message is sent by directly using the instance of the endpoint
+## Outside the pipeline
```

Comment:
Rather than outside the pipeline and inside the pipeline, for a user, it is much easier to understand it in terms of a message handler. 
Consider talking in terms of sending a message from within a message handler and outside of a message handler. It is much much easier to relate than pipeline. Pipeline is more for advanced users. Sending a message should cater to the basic user.